### PR TITLE
🌱 Fixes typo in machine types description

### PR DESCRIPTION
**Pull request by srm09**
**What this PR does / why we need it**:
nit: Fixes a typo in comment

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
n/a

### CODE REVIEW

File: api/v1alpha4/machine_types.go [hunk 1]
```diff
@@ -210,7 +210,7 @@ func (m *MachineStatus) GetTypedPhase() MachinePhase {
 
 // ANCHOR: Bootstrap
 
-// Bootstrap capsulates fields to configure the Machine’s bootstrapping mechanism.
+// Bootstrap encapsulates fields to configure the Machine’s bootstrapping mechanism.
```
Comment (CecileRobertMichon) [inline api/v1alpha4/machine_types.go:213]: I think you need to run `make generate` to actually update the description in config/

Comment (fabriziopandini) [reply api/v1alpha4/machine_types.go:213]: Uhmm this should be captured by verify scripts...

Comment (vincepri) [reply api/v1alpha4/machine_types.go:213]: I don't think the struct comment is actually used anywhere, the description of the field (when using a struct as a pointer or not) is derived from the comment on that field, not the one on the linked struct. 

Tested the verify script locally and it's still working properly.

**Review by vincepri — APPROVED** [review]
/lgtm

**Review by CecileRobertMichon — COMMENTED** [review]
/approve

**Pull request merged**